tools/clang/test/HLSLFileCheck/hlsl/intrinsics/wave/reduction/WaveAndBreakLib.hlsl

```diff
@@ -4,6 +4,8 @@ StructuredBuffer<int> buf[]: register(t2);
 
 // Cannonical example. Expected to keep the block in loop
 // Verify this function loads the global
+// CHECK: define i32
+// CHECK-SAME: WaveInLoop
```

Comment:
Stole these from my other pending dx.break change to make errors clearer by preventing later functions from satisfying CHECK criteria.